support/cas-server-support-jdbc-drivers/src/main/java/org/apereo/cas/JdbcServletContextListener.java

```diff
@@ -0,0 +1,53 @@
+package org.apereo.cas;
+
+import lombok.val;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.logging.Logger;
+
+/**
+ * This is {@link JdbcServletContextListener} that properly
+ * deregisters JDBC drivers.
+ *
+ * @author leeyc0
+ * @since 6.2.0
+ */
+@WebListener
+public class JdbcServletContextListener implements ServletContextListener {
+
+    @Override
+    public void contextInitialized(final ServletContextEvent sce) {
+    }
+
+    @Override
+    public final void contextDestroyed(final ServletContextEvent sce) {
+        /* Slf4j does not work in contextDestroyed */
```

Comment:
This should work.

Reply:
Atual I tried Slf4j first and did not work.

Reply:
It appears that Slf4j logger already closed before contextDestroyed is called.